pkg/processor/reportprocessor.go

```diff
@@ -213,9 +213,14 @@ func getGovernanceResults(dynamicClient dynamic.Interface, clusterInfo types.Man
 			glog.Warningf("error parsing status details as a map for policy %s: %s", plcName, err)
 			continue
 		}
+		compliance, _, err := unstructured.NestedString(status, "compliant")
```

Comment:
Suggestion:
Maybe we could reduce memory if we avoid the intermediate `status` object. I don't know if it will be noticeable at all, but should help with large data.
```suggestion
		compliance, _, err := unstructured.NestedString(plc.Object, "status", "compliant")
```

Same for `status.details` above.

Reply:
That'd be cleaner, too. I'll update it--thanks!